src/main/java/org/hobbit/utils/kubernetes/KubeHelper.java

```diff
@@ -0,0 +1,40 @@
+package org.hobbit.utils.kubernetes;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+public class KubeHelper {
+    private static final Logger LOGGER = LoggerFactory.getLogger(KubeHelper.class);
+    /**
+     * Gets dns friendly ip
+     *
+     * @return plain hostname (or address)
+     */
+    public static String getDnsFriendlyIP(String namespace) {
```

Comment:
`getDNSName`

Reply:
I think the method name defines the functionality of the method better, so maybe is better to keep it